docs/QSG/qemu-vms.md

```diff
@@ -0,0 +1,285 @@
+```
+    sudo qemu-system-x86_64 -M q35,accel=kvm,kernel-irqchip=split -cpu host -smp 3 -m 12288 -device intel-iommu,intremap=on -drive file=<image-dir>/daos-server.qcow2,if=virtio -device virtio-net,netdev=mynet0,mac=52:54:00:12:34:56 -netdev tap,id=mynet0 -drive file=<image-dir>/qemu-nvm-disk1.qcow2,if=none,id=nvm1 -device nvme,serial=deadbeef,drive=nvm1 &
+    sudo qemu-system-x86_64  -M q35,accel=kvm,kernel-irqchip=split -cpu host -smp 1 -m 2048 -device intel-iommu,intremap=on -drive file=<image-dir>/daos-client.qcow2,if=virtio -device virtio-net,netdev=mynet1,mac=52:54:00:12:34:57 -netdev tap,id=mynet1 
+```
+I follow these [steps](https://docs.daos.io/latest/QSG/setup_rhel/) to install both the DAOS server, DAOS admin, and DAOS client. I install daos-server and daos-admin on the first VM, and install daos-client on the second VM. If you have issues with Ceph Pacific Repo you can edit `/etc/yum.repos.d/CentOS-Ceph-Pacific.repo` by commenting out mirrorlist and changing baseurl as below.
+```
+...
+[centos-ceph-pacific]
+name=CentOS-$stream - Ceph Pacific
+#mirrorlist=http://mirrorlist.centos.org/?release=$stream&arch=$basearch&repo=storage-ceph-pacific
+baseurl=http://buildlogs.centos.org/centos/$stream/storage/$basearch/ceph-pacific/
+gpgcheck=1
+enabled=1
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-SIG-Storage
+...
+```
+After DAOS installation you still need to update the config files.
+
+5.  Update config files.
+
+Update the daos-server config file `/etc/daos/daos_server.yml` on daos-server. You may need to change fabric\_iface and bdev\_list if they are different.
+```
+    name: daos_server
+    access_points:
+    - daos-server
+    port: 10001
+    
+    nr_hugepages: 1024
+    system_ram_reserved: 4
+    disable_vfio: true 
+    
+    transport_config:
+        allow_insecure: false
+        client_cert_dir: /etc/daos/certs/clients
+        ca_cert: /etc/daos/certs/daosCA.crt
+        cert: /etc/daos/certs/server.crt
+        key: /etc/daos/certs/server.key
+    # Haven't got ofi+tcp to work with QEMU. Need further investigation.
+    provider: ofi+sockets
+    control_log_mask: DEBUG
+    control_log_file: /tmp/daos_server.log
+    helper_log_file: /tmp/daos_server_helper.log
+    engines:
+    -
+        targets: 1
+        pinned_numa_node: 0
+        nr_xs_helpers: 0
+        fabric_iface: enp0s2
+        fabric_iface_port: 31316
+        log_mask: DEBUG
+        log_file: /tmp/daos_engine_0.log
+        storage:
+        -
+            class: ram
+            scm_mount: /mnt/daos0
+            scm_size: 4
+        -
+            class: nvme
+            bdev_list:
+            - "0000:00:03.0"
+```
+Because we disable VFIO and use UIO, we need to run daos<sub>server.service</sub> as root. Open /usr/lib/systemd/system/daos<sub>server.service</sub>, then change User and Group to root. Then run `systemctl daemon-reload` to reload systemd scripts.
```

Comment:
fix the markdown syntac for daos_server.service and also 

/usr/lib/systemd/system/daos_server.service